src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathFilter.java

```diff
@@ -90,6 +87,9 @@
  * JavadocBlockTagLocation
  * </li>
  * <li>
+ * JavadocMethod
+ * </li>
+ * <li>
```

Comment:
Please clarify why did you move it here?

Reply:
reason is the 1. I refer to PR #8214 which add it here. 2. As mentioned in https://github.com/checkstyle/checkstyle/issues/7739#issuecomment-611372244 the 1,3,6th log within the check is logging the tags inside the javadoc block comment. It is not accessible using DetailAST. Maybe this is reason to add to here.
https://github.com/checkstyle/checkstyle/blob/d48b815db7c6e9a9272ec78fc7d2cec0d53a8e26/src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathFilter.java#L74-L114

Reply:
I don't understand your response. Here, you put `<li>JavadocMethod</li>` after `<li>JavadocBlockTagLocation</li>`. But before your changes the `<li>JavadocMethod</li>` was few lines upper, after `<li>Indentation</li>`. So, what's the reason for this movement?

Reply:
Ok, I got it. The `<ul>` element is now different.